packages/node/src/lib/organizations/organizations.interface.ts

```diff
@@ -0,0 +1,34 @@
+export interface IOrganizations {
+  list();
+  create(payload: IOrganizationCreatePayload);
+  rename(payload: IOrganizationRenamePayload);
+  getCurrent();
+  removeMember(memberId: string);
+  updateMemberRole(
+    memberId: string,
+    payload: IOrganizationUpdateMemberRolePayload
+  );
```

Comment:
These two methods also may cause problems when full Role-based access control will be implemented?

I can imagine member that may bypass role-based access control by using API KEY to remove other members or update their role - they will do it as "organization owner" (assuming that member would be able to see api key).